packages/tracing/src/utils.ts

```diff
@@ -41,9 +72,56 @@ export function extractTraceparentData(traceparent: string): TraceparentData | u
       parentSpanId: matches[2],
     };
   }
+
   return undefined;
 }
 
+type TracestateHeaderData = { sentry?: string; thirdparty?: string };
+
+/**
+ * Extract data from an incoming `tracestate` header
+ *
+ * @param header
+ * @returns Object containing data from the header
+ */
+export function extractTracestateData(header: string): TracestateHeaderData {
+  let sentryEntry, thirdPartyEntry;
+
+  if (header) {
```

Comment:
This check is unnecessary. You can exec on empty string just fine.

Reply:
Yeah, and I guess we do check for the header's existence before calling this in all of the spots where we use it. Okay, removing.